.github/workflows/php-tests.yml

```
@@ -41,9 +41,18 @@ jobs:
       # with coverage of PHP 7.4, 8.2, and 8.5.
+      #
+      # WP backlevel is L-1, not L-2: every released WC major in our matrix
+      # (the L and L-2 cells) requires WP >= 6.8, and WP L-2 is 6.7. Pinning
+      # WP=L-2 here makes WC's installer reject the WP version and the
+      # whole job fails before PHPUnit runs. Once a future WC major lifts
+      # its WP minimum past 6.9 we can revisit, but until WC ships a
+      # release that supports WP L-2 again we have no way to honour the
+      # strict L-2 policy from AGENTS.md without rolling WC back further
+      # than the loose L-2 policy allows.
       matrix:
         woocommerce_support_policy: [ 'L', 'L-2' ]
```

YAMLlint (1.38.0)

[error] too many spaces inside brackets

(brackets)